css/style.css

```diff
@@ -0,0 +1,200 @@
+:root{
+    --color-nav: #be3144;
+    --color-project:#45567d;
+    --color-white: #f0f0f0;
+    --color-grey: #303841;
+
+    /* shadows */
+    --shadow-nav: 0 2px 1 rgba(0, 0, 0, .4);
+    --shadow-project: 1px 1px 2px rgba(0, 0, 0, .5);
+    --shadow-text: 2px 2px 1px #1f1f1f;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+*,
+*::before,*::after {
+    box-sizing: inherit;
+}
+
+html {
+    font-size: 62.5%; 
+}
+
+body {
+    font-family: sans-serif;
+    font-size: 1.8rem;
+    font-weight: 400;
+    line-height: 1.4;
+    color: var(--color-white);
+}
+
+.nav {
+    display: flex;
+    justify-content: flex-end;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    background-color: var(--color-nav);
+    z-index: 100;
+    box-shadow: var(--shadow-nav);
+}
+
+.nav-list {
+    display: flex;
+    margin-right: 2rem;
+}
+
+.nav-list a {
+    display: block;
+    padding: 2rem;
+    font-size: 2.2rem;
+}
+
+a {
+    text-decoration: none;
+    color: var(--color-white);
+}
+
+a:link {
+    cursor: pointer;
+}
+
+ul {
+    list-style: none;
+}
+
+li {
+    display: list-item;
+    text-align: center;    
+}
+
+.welcome-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100vh;
+    background-color: #000;
+    background-image: linear-gradient(62deg, #3a3d40 0%, #181719 100%);
+}
+
+h1 {
```

Comment:
- [ ] Please try to make the students understand the difference between `tag name` and  `class attribute` and when to use them appropriately.

Reply:
Please use class attributes like `.title` rather than using tag names like h1, using an appropriate class name in CSS will make your code easier to read and maintain.